HousingRepairsOnlineApi/Helpers/RepairExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.UseCases;
+
+
+namespace HousingRepairsOnlineApi.Helpers
```

Comment:
To aid discoverability of extension methods on Repair, the namespace should be the same as the class being extended, i.e. 
```suggestion
namespace HousingRepairsOnlineApi.Domain
```

On that basis, it would also be best to move this file to the `Domain` folder too.